dotcom-rendering/src/experiments/tests/sign-in-gate-main-variant.ts

```diff
@@ -7,7 +7,7 @@ export const signInGateMainVariant: ABTest = {
 	author: 'Mahesh Makani',
 	description:
 		'Show sign in gate to 100% of users on 3rd article view of simple article templates, and show a further 5 times after the first dismissal, with higher priority over banners and epic. Main/Variant Audience.',
-	audience: 0.75,
+	audience: 0.9,
```

Comment:
Is it intentional to increase the audience in this PR?

Reply:
Yep - in removing the temporary test, we need to restore the standard test back to 90% coverage. Should I do that in a separate PR?

Reply:
No, all good, just wanted to check it was a deliberate change 👍